cmd/jujud/agent/caasoperator/manifolds.go

```diff
@@ -164,18 +164,18 @@ func Manifolds(config ManifoldsConfig) dependency.Manifolds {
 		// controls the messages sent via the log sender according to
 		// changes in environment config. We should only need one of
 		// these in a consolidated agent.
-		// loggingConfigUpdaterName: ifNotMigrating(logger.Manifold(logger.ManifoldConfig{
-		// 	AgentName:     agentName,
-		// 	APICallerName: apiCallerName,
-		// })),
+		loggingConfigUpdaterName: ifNotMigrating(logger.Manifold(logger.ManifoldConfig{
```

Comment:
Does `ifNotMigrating` even work for a CAAS model? I think that should go. We certainly don't care about migrations for the prototype.